docs/01-overview/02-main-areas/observability/obsv-10-useful-links.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Observability - Useful links
+---
+
+If you're interested in learning more about the Observability area, check out these links:
+
+- Learn how to set up the [Monitoring Flow](../../../03-tutorials/observability/obsv-01-monitoring-overview.md) for your services in Kyma, in which you [observe your application metrics](../../../03-tutorials/observability/obsv-02-observe-application-metrics.md), [set up your Grafana Dashboard](../../../03-tutorials/observability/obsv-03-create-and-configure-grafana-dashboard.md), [define alerting rules](../../../03-tutorials/observability/obsv-04-define-alerting-rules-monitor.md), and eventually, [send notifications](../../../03-tutorials/observability/obsv-05-send-notifications.md). ...And remember to [clean up when you're done](../../../03-tutorials/observability/obsv-06-clean-up-configuration.md) ;)
+
+- Check out the different ways to [access the logs](../../../04-operation-guides/operations/obsv-01-access-logs.md) provided by Kubernetes and Loki.
+
+- See how to [access and expose](../../../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana.md) the observability-related services Grafana<!-- , Jaeger, --> and Kiali.
+
+- Find out what to do when the [Prometheus Istio Server keeps crashing](../../../04-operation-guides/troubleshooting/obsv-01-troubleshoot-prometheus-istio-server-crash-oom.md) or when [Jaeger shows fewer traces than expected](../../../04-operation-guides/troubleshooting/obsv-02-troubleshoot-jaeger-shows-few-traces.md).
+
+- Understand the architecture of Kyma's [monitoring](../../../05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md), [logging](../../../05-technical-reference/03-architecture/obsv-02-architecture-logging.md), and [tracing](../../../05-technical-reference/03-architecture/obsv-03-architecture-tracing.md) components.
```

Comment:
@a-thaler should we update this to reflect the prometheus istio server too ? Here the point says Kymas monotoring arch I think it should also be shown ?

Reply:
I believe if we regard "Observability" as a proper noun (aka for the Kyma main area), we should drop the hyphen. If we use it as a normal noun as a concept of software or systems, we shouldn't capitalize. I'm fine with both.